src/Dfe.PlanTech.Web/Views/Shared/Components/VerticalNavigation/Default.cshtml

```diff
@@ -1,17 +1,21 @@
 @model IEnumerable<Dfe.PlanTech.Domain.Questionnaire.Interfaces.IHeaderWithContent>
 
 <nav class="dfe-vertical-nav">
+    <div class="dfe-vertical-nav__back-button govuk-!-margin-bottom-5">
+        <span> &lt; </span>
```

Comment:
I'd have expected this to come from a css class rather than &lt - can we not copy what back button does?

Reply:
I forgot that the back link looked like that :) I did try to add the `<` with a `::before` style and some other fancy jazz but then gave up and this seemed good.

But now you've pointed that out, I've just used the back link class. Good shout!